runtime/js.rs

```diff
@@ -35,8 +34,5 @@ pub fn deno_isolate_init() -> Snapshot {
   Snapshot::Static(&RUNTIME_SNAPSHOT)
 }
 
-pub fn get_99_main() -> PathBuf {
-  let manifest = env!("CARGO_MANIFEST_DIR");
-  let path = PathBuf::from(manifest);
-  path.join("js").join("99_main.js")
-}
+#[cfg(feature = "snapshot_from_snapshot")]
+pub static SOURCE_CORE_FOR_99_MAIN_JS: &str = include_str!("js/99_main.js");
```

Comment:
this data seems to only be used in build.rs... why not put it there? it seems the contents of 99_main.js will be in the runtime's memory

Reply:
It's used in `cli/build.rs` and this code lives in `runtime/` - there's no way I'm aware of of `include_str!()` file from other crate (because you need to pass a string literal to `include_str!()`. `#[cfg(feature="snapshot_from_snapshot")]` ensures that this source is only embedded when you are snapshotting.

Reply:
Maybe add a comment about that... seems a bit odd to me.

Reply:
I will try to clean this up further in https://github.com/denoland/deno/pull/17653 so we don't end up with two copies of this source code in the binary (one in data section, the other in the snapshot).